accord-core/src/main/java/accord/local/RedundantBefore.java

```diff
@@ -109,6 +109,8 @@ public Entry(Range range, long startEpoch, long endEpoch, @Nonnull TxnId locally
             this.shardAppliedOrInvalidatedBefore = shardAppliedOrInvalidatedBefore;
             this.bootstrappedAt = bootstrappedAt;
             this.staleUntilAtLeast = staleUntilAtLeast;
+            Invariants.checkArgument(locallyAppliedOrInvalidatedBefore.equals(TxnId.NONE) || locallyAppliedOrInvalidatedBefore.domain().isRange());
```

Comment:
so this just blocks key txn from showing up?  In BurnTest we can do `Range` read/write and I think we wanted to add range reads in C* eventually, so is `range` the best here or is it really `ExclusiveSyncPoint` that we want?

Reply:
We shouldn’t be inserting them into this map is all - this is a map for GC and we don’t want that to be at the point/key level